runtime/lua/vim/lsp/buf.lua

```diff
@@ -198,9 +190,13 @@ end
 ---
 ---@see vim.lsp.protocol.CompletionTriggerKind
 function M.completion(context)
```

Comment:
Also potential follow up: Deprecate this in favour of lsp.completion.trigger

Reply:
#30933